src/components/Cheese/CheeseCircle.css

```diff
@@ -19,11 +19,11 @@
 }
 /*.front svg > circle {
   stroke: var(--primaryLinkColor) !important;
-}*/
+}
 .back svg > circle {
   stroke: rgba(255, 255, 255, .06) !important;
 }
-
+*/
```

Comment:
`.back` making gray background until the goal is fully achieved. Without it would seem that the ring is completely filled. `.front svg > circle` can be removed since color already set https://github.com/odota/ui/blob/master/src/components/App/App.jsx#L22